app/routes.py

```diff
@@ -9,21 +9,38 @@ def get_posts():
 
 
 @app.route('/addPost',methods=['POST'])
-def add_post():
+def addPost():
     post_data=request.get_json()
     title=post_data.get('title')
     body=post_data.get('body')
 
     post=Post(title,body)
     post.save()
-    return "done"
+    return jsonify({"message":"Successfully added"})
 
-@app.route('/deletePost',methods=['POST'])
-def delete_post():
-    post_data = request.get_json()
-    title=post_data.get('title')
-    body=post_data.get('body')
-    post=Post.query.filter_by(title=title).first()
-    post.delete()
-    return "done"
+@app.route('/deletePost/<post_id>',methods=['DELETE'])

```

Comment:
As per the naming convention this should be 

```suggestion
@app.route('/posts/<post_id>',methods=['DELETE'])

```